jobs/rootfs-builder.jpl

```diff
@@ -80,7 +95,14 @@ node("debos && docker") {
     def config = "${params.ROOTFS_CONFIG}"
     def arch = "${params.ROOTFS_ARCH}"
     def pipeline_version =  "${params.PIPELINE_VERSION}"
-    def docker_image = "${params.DOCKER_BASE}debos"
+    def docker_image = " "
+    def rootfs_type = "${params.ROOTFS_TYPE}"
+
+    if (rootfs_type == "buildroot") {
+        docker_image = "${params.DOCKER_BASE}buildroot"
+    } else {
```

Comment:
I think it's better to explicitly use `else if (rootfs_type == "debos")` instead of  `else`.
We may have more rootfs types in future and `else` may lead to false positives.